pdf/contentstream/draw/shapes.go

```diff
@@ -175,6 +175,14 @@ const (
 	LineEndingStyleButt  LineEndingStyle = 2
 )
 
+// LineStyle refers to how the line will be created
```

Comment:
all godoc comments should be a sentence ending with a period (.)
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
e.g. // LineStyle defines the drawing style used when stroking a line.